include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
     
     /* Extensions Exposed as Features: */
 
+    /* VK_AMD_buffer_marker */
+    /* VK_AMD_calibrated_timestamps */
+    /* VK_AMD_display_native_hdr */
+    /* VK_AMD_draw_indirect_count */
+    /* VK_AMD_gcn_shader */
+    /* VK_AMD_gpa_interface */
+    /* VK_AMD_gpu_shader_half_float */
+    /* VK_AMD_gpu_shader_half_float_fetch */
+    /* VK_AMD_gpu_shader_int16 */
+    /* VK_AMD_image_layout_resolve */
+    /* VK_AMD_memory_overallocation_behavior */
+    /* VK_AMD_mixed_attachment_samples */
+    /* VK_AMD_negative_viewport_height */
+    /* VK_AMD_pipeline_compiler_control */
+    /* VK_AMD_programmable_sample_locations */
+    /* VK_AMD_rasterization_order */
+    /* VK_AMD_shader_ballot */
+    /* VK_AMD_shader_explicit_vertex_parameter */
+    /* VK_AMD_shader_fragment_mask */
+    /* VK_AMD_shader_image_load_store_lod */
+    /* VK_AMD_shader_info */
+    /* VK_AMD_shader_trinary_minmax */
+    /* VK_AMD_texture_gather_bias_lod */
+    /* VK_AMD_wave_limits */
+    /* VK_ANDROID_external_memory_android_hardware_buffer */
+    /* VK_ANDROID_native_buffer */
+    /* VK_EXT_2d_imageview_3d_image */
+    /* VK_EXT_calibrated_timestamps */
+    /* VK_EXT_debug_marker */
```

Comment:
cool extension, Nahim should implement it for commandbuffers, will help with "annotating" "snapshot points" at which resources from 2 renderdoc captures should be compared.